inlong-sort/sort-flink/sort-flink-v1.15/pom.xml

```diff
@@ -0,0 +1,228 @@
+                <version>${flink.pulsar.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.ververica</groupId>
+                <artifactId>flink-connector-mongodb-cdc</artifactId>
+                <version>${flink.connector.mongodb.cdc.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.bahir</groupId>
+                <artifactId>flink-connector-redis_${flink.scala.binary.version}</artifactId>
+                <version>${flink.connector.redis}</version>
+            </dependency>
+            <!-- hudi -->
+            <dependency>
+                <groupId>org.apache.hudi</groupId>
+                <artifactId>hudi-flink1.13-bundle</artifactId>
+                <version>${hudi.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.streamnative.connectors</groupId>
+                <artifactId>flink-protobuf</artifactId>
+                <version>${flink.protobuf.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.microsoft.sqlserver</groupId>
+                <artifactId>mssql-jdbc</artifactId>
+                <version>${sqlserver.jdbc.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.iceberg</groupId>
+                <artifactId>iceberg-flink-runtime-1.14</artifactId>
+                <version>${iceberg.version}</version>
+            </dependency>
+
+            <!-- flink -->
+            <dependency>
+                <groupId>org.apache.flink</groupId>
+                <artifactId>flink-table-api-java-bridge</artifactId>
+                <version>${flink.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.flink</groupId>
+                <artifactId>flink-table-runtime</artifactId>
+                <version>${flink.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.flink</groupId>
+                <artifactId>flink-clients</artifactId>
+                <version>${flink.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.flink</groupId>
+                <artifactId>flink-table-planner_${flink.scala.binary.version}</artifactId>
+                <version>${flink.version}</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.flink</groupId>
+                <artifactId>flink-streaming-java</artifactId>
+                <version>${flink.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.flink</groupId>
+                <artifactId>flink-streaming-scala_${flink.scala.binary.version}</artifactId>
+                <version>${flink.version}</version>
+            </dependency>
```

Comment:
flink core code scope should be provided

Reply:
done

Reply:
> done

@GanfengTan Maybe flink 1.13 module have same problem

Reply:
<img width="776" alt="image" src="https://github.com/apache/inlong/assets/17596132/374afc0f-8d03-4249-97e6-63f144fadb17">
Dependency transitive relationship.

Reply:
@GanfengTan I see, I missed  a pom file because of too many dir.